content/en/blog/2021-04-29-may-update.md

```diff
@@ -63,7 +63,7 @@ The Flux v2 CLI and the GitOps Toolkit controllers are now CII Best
 Practices certified.
 
 Checkout the [new bootstrap
-procedure](/docs/guides/installation/#generic-git-server).
+procedure](/docs/installation/_index.md#generic-git-server).
```

Comment:
General question: do you envision more docs under `/installation/`?

Reply:
Yes, that page is already too big and we'll need to add more stuff to it.

Reply:
Super!